src/renderer/components/virtual-list/virtual-list.scss

```diff
@@ -2,12 +2,6 @@
   overflow: hidden;
 
   > .list {
-    @include custom-scrollbar;
-
-    .theme-light & {
-      @include custom-scrollbar(dark);
-    }
-
-    overflow-y: overlay !important;
+    overflow-y: overlay!important;
```

Comment:
Why `!important`?

Reply:
It overrides `overflow: auto` inline styles defined by `VirtualList` component.

Reply:
Or by `react-window` internals did you mean?